packages/react/src/box/styles.ts

```diff
@@ -555,6 +555,11 @@ export const linkStyles = {
 		'&[href]::after': {
 			content: '" (" attr(href) ")" !important',
 		},
+
+		// Hide internal hash link URLs
+		'&[href^="#"]:after': {
```

Comment:
pseudo elements are different than pseudo classes
```suggestion
		'&[href^="#"]::after': {
```

Reply:
Updated to match pseudo-element

https://github.com/agriculturegovau/agds-next/pull/1970/commits/8071497027261d563134d5c2faba3939192b261e